streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1164,8 +1169,30 @@ public void cleanUp() {
      * {@code queryableStoreType} doesn't exist
      */
     public <T> T store(final String storeName, final QueryableStoreType<T> queryableStoreType) {
-        validateIsRunning();
-        return queryableStoreProvider.getStore(storeName, queryableStoreType);
+        return store(storeName, queryableStoreType, false);
+    }
+
+    /**
+     * Get a facade wrapping the local {@link StateStore} instances with the provided {@code storeName} if the Store's
+     * type is accepted by the provided {@link QueryableStoreType#accepts(StateStore) queryableStoreType}.
+     * The returned object can be used to query the {@link StateStore} instances.
+     *
+     * @param storeName           name of the store to find
+     * @param queryableStoreType  accept only stores that are accepted by {@link QueryableStoreType#accepts(StateStore)}
+     * @param queryStaleData      If false, only permit queries on the leader replica for a partition, and only if the
```

Comment:
should we use `active` instead of `leader` to stay consistent with streams terminology?